docs/GraphQL-API/query/datetime.md

```diff
@@ -6,44 +6,44 @@ Datetime object has connections with `RootQuery`, `Event`, `Ticket` etc. and thu
 
 ```gql
 query getDatetimes($first: Int, $where: RootQueryDatetimesConnectionWhereArgs) {
-  datetimes(first: $first, where: $where) {
-    edges {
-      node {
-        id
-        name
-        description
-        sold
-        reserved
-        order
-        startDate
-        endDate
-        startTime
-        endTime
-      }
-    }
-  }
+	datetimes(first: $first, where: $where) {
+		edges {
+			node {
+				id
+				name
+				description
+				sold
+				reserved
+				order
+				startDate
+				endDate
+				startTime
+				endTime
+			}
+		}
+	}
```

Comment:
Prettier is working :fireworks:

Reply:
Not the way it's intended to work 😞 
We don't need tabs in md files 🙄

Reply:
It might need some additional configuration for tabs.
But it works great in other situations with `.md` files:
![Peek 2019-12-04 14-22](https://user-images.githubusercontent.com/1477453/70142364-bc204a80-16a1-11ea-9520-fcff97b9c428.gif)